### ISSUE TITLE
SDXL

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#63]
### Source / repo
https://huggingface.co/docs/diffusers/using-diffusers/sdxl (placeholder for now)

### Model description
[DESCRIPTION]

### Dataset
[DATASET]

### Literature benchmark source
[URL]

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

### Logs gist
[URL]
